package.json

```diff
@@ -26,9 +26,11 @@
     "bump:alpha": "lerna version prerelease --preid $npm_config_preid --no-changelog --exact --yes --no-private",
     "publish:alpha": "lerna publish from-package --dist-tag alpha --no-verify-access --yes",
     "bump:production": "lerna version --conventional-graduate --exact --no-private",
-    "publish:production": "PUBLIC_SM_UI_SEGMENT_KEY=cGjidifKefYb6EPaGaqpt8rQXkv5TD6P PUBLIC_SM_INIT_SEGMENT_KEY=ED3O1FVsa8DteNyRmAuDyMcPoftJ6VdP lerna publish from-package"
+    "publish:production": "PUBLIC_SM_UI_SEGMENT_KEY=cGjidifKefYb6EPaGaqpt8rQXkv5TD6P PUBLIC_SM_INIT_SEGMENT_KEY=ED3O1FVsa8DteNyRmAuDyMcPoftJ6VdP lerna publish from-package",
+    "depcheck": "depcheck --skip-missing; lerna run depcheck"
```

Comment:
why the use of skip missing if this is in the config already ?

Reply:
I didn't add a config to the root since there's nothing to ignore. So just left the flag

Reply:
ah ok 👍